Build broken with GDAL 3

Build fails hard on the import/export gdal code:
```
../../src/ImportExport/ImportExportGdal.cpp:114:54: error: assigning to 'OGRSFDriver *' from incompatible type 'GDALDriver *'
    poDriver = OGRSFDriverRegistrar::GetRegistrar()->GetDriverByName(pszDriverName);
../../src/ImportExport/ImportExportGdal.cpp:493:34: error: no member named 'Open' in 'OGRSFDriverRegistrar'
    poDS = OGRSFDriverRegistrar::Open( FileName.toUtf8().constData(), FALSE );
           ~~~~~~~~~~~~~~~~~~~~~~^
../../src/ImportExport/ImportExportGdal.cpp:520:34: error: no member named 'Open' in 'OGRSFDriverRegistrar'
    poDS = OGRSFDriverRegistrar::Open( "/vsimem/temp", FALSE );
```

--- a/src/Main.cpp
+++ b/src/Main.cpp
@@ -9,6 +9,9 @@
 #include <qtsingleapplication.h>
 #include "MainWindow.h"
 #include "Preferences/MerkaartorPreferences.h"
+#ifndef ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#endif
 #include "proj_api.h"
 #include "gdal_version.h"
 #include "Global.h"

--- a/src/MainWindow.cpp
+++ b/src/MainWindow.cpp
@@ -102,6 +102,9 @@
 #include <limits.h>
 
 //For About
+#ifndef ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#endif
 #include "proj_api.h"
 #include "gdal_version.h"
 

--- a/src/common/Projection.h
+++ b/src/common/Projection.h
@@ -9,6 +9,9 @@
 #ifndef _MOBILE
 #include "MerkaartorPreferences.h"
 
+#ifndef ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#endif
 #include <proj_api.h>
 typedef projPJ ProjProjection;
 

